javascript/operators/import.json

```diff
@@ -65,6 +65,65 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "options": {
+          "__compat": {
+            "description": "The second options parameter",
```

Comment:
I called it "options" instead of "assertions", because unlike the `import` statement, the options here permit any unknown attributes from the start, so other keys won't throw parser errors either. In this case, should "assertions" be separately listed as a subfeature to increase discoverability?